events/wc_druidic_events.txt

```diff
@@ -148,7 +210,108 @@ narrative_event = {
 		}
 	}
 }
-# Suggests liege use worgens against enemies
+# Arugal returns to Gilneas, or not...
+narrative_event = {
+	id = WCDRU.304
+	title = EVTTITLE_WCDRU_304
+	desc = EVTDESC_WCDRU_304
+	picture = GFX_evt_dalaran
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+
+	c_46055 = yes
```

Comment:
Wrong syntax? Never seen it being pre-trigger.
https://ck2.paradoxwikis.com/Event_modding#Pre-triggers

Reply:
I'm not familiar with pre-triggers, so no surprise it doesn't work there. When modding I generally stayed away from events.

has_character_flag looks like the only way to limit an event check to a specific character, but there doesn't seem to be a way of setting the flag in the character file, and using an event would defeat the point. An I am Arugal decision sounds stupid, but I guess it would work if there's no other option.

Reply:
>but there doesn't seem to be a way of setting the flag in the character file

Check Ragnarr Lodbrok character file and events

```
766.6.6 = {
	birth = yes
	effect = { set_character_flag = is_ragnar_lodbrok }
}